joomla/joomla-cms
# [4.0] blog.php Scrabble96 patch 4 - featured

Pull Request for Issue # .
#26160 
REQUIRES #26231, #26242 
### Summary of Changes

Removed IE support per #26160 

### Testing Instructions
Create or view a category blog.
View Source
blog-item-content class should be gone.

### Expected result
![image](https://user-images.githubusercontent.com/1850089/64564207-9e4b7f80-d316-11e9-8bb3-08df77c69bdd.png)



### Actual result
![image](https://user-images.githubusercontent.com/1850089/64564103-6f350e00-d316-11e9-8559-e569dc7b9eb3.png)



### Documentation Changes Required
